src/components/posts/PostItem.js

```diff
@@ -1,17 +1,20 @@
 import classes from "./PostItem.module.css";
 import Post from "../elements/Post";
 import LikeButton from "../elements/LikeButton";
+import EditableDiv from '../elements/EditableDiv';
 
 function PostItem(props) {
   return (
     <Post>
       <div className={classes.containerHeader}>
         <span className={classes.author}>{props.author}</span>
         <span>{props.likes}</span>
-        <LikeButton id={props.id}/>
+        <LikeButton id={props.id} />
```

Comment:
I see some self closing elements have a space and some don't, it's better to be consistent: which one do you want to stick with?

Reply:
I believe anytime I don't use any innerHTML I've used self closing elements - will change anytime an exception to this.